components/proxy/src/test/java/com/hotels/styx/proxy/plugin/InstrumentedPluginTest.java

```diff
@@ -76,128 +72,132 @@ public void setUp() {
     @Test
     public void metricIsRecordedWhenResponseIsMappedToErrorStatus() {
         Chain chain = request -> aResponse(OK);
-
-        InstrumentedPlugin plugin = instrumentedPlugin("replaceStatusCode", (request, aChain) ->
+        String pluginName = "replaceStatus1";
+        InstrumentedPlugin plugin = instrumentedPlugin(pluginName, (request, aChain) ->
                 aChain.proceed(request)
                         .map(response -> responseWithNewStatusCode(response, INTERNAL_SERVER_ERROR)));
 
         LiveHttpResponse response = Mono.from(plugin.intercept(someRequest, chain)).block();
 
         assertThat(response.status(), is(INTERNAL_SERVER_ERROR));
-        assertThat(metricRegistry.meter("plugins.replaceStatusCode.response.status.500").getCount(), is(1L));
-        assertThat(metricRegistry.meter("plugins.replaceStatusCode.errors").getCount(), is(1L));
+
+        assertThat(Metrics.counter("plugins.response.status", "plugin", pluginName, "status", "500").count(), is(1.0));
```

Comment:
You can retrieve the counter from your metricRegistry with `metricRegistry.find(name).tags(tags).counter()`. It's a bit more long-winded, but then you don't need to link your registry with the global registry in `InstrumentedPlugin`